st2common/tests/unit/test_paramiko_script_action_model.py

```diff
@@ -37,10 +37,11 @@ def test_get_command_string(self):
                                                    parallel=True,
                                                    sudo=False,
                                                    timeout=60)
-        expected = '/tmp/remote_script.sh song=\'b s\' "taylor swift"'
+        expected = '/tmp/remote_script.sh song=\'b s\' \'taylor swift\''
         self.assertEqual(script_action.get_full_command_string(), expected)
 
         # Test with sudo
         script_action.sudo = True
-        ex = 'sudo -E -- bash -c \'/tmp/remote_script.sh song=\'"\'"\'b s\'"\'"\' "taylor swift"\''
+        ex = 'sudo -E -- bash -c ' + \
+             '\'/tmp/remote_script.sh song=\'"\'"\'b s\'"\'"\' \'"\'"\'taylor swift\'"\'"\'\''
```

Comment:
Kinda skeptic whether this is fine. We need to write a sudo action that actually takes params with spaces (both positional and named args) and validate that args are passed in correctly. I'd be happy if we added this test in st2tests. cc: m4dcoder.